examples/interactive_play_human.py

```diff
@@ -0,0 +1,947 @@
+#!/usr/bin/env python3
```

Comment:
I think that we should refrain from adding these new `interactive_play_*.py` scripts to main.

It may be preferable to keep them on separate branches for reference.